cmd/easeprobe/main.go

```diff
@@ -43,8 +44,30 @@ func getEnvOrDefault(key, defaultValue string) string {
 	return defaultValue
 }
 
-func main() {
+func showVersion() {
+
+	var v = global.Ver
+
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		v = fmt.Sprintf("%v %v", global.DefaultProg, v)
+		fmt.Println(v)
+	}
```

Comment:
this is the old ver format without gitsha, otherwise -v returns nothing looks weird 

i do not know when will it be `false`, seems not being built in git repo

Reply:
if the `debug.ReadBuildInfo()` return the `false`, I believe the `bi` is nil, if you don't return here, the following code would be panic with the nil pointer access error. right?

Reply:
good catch
mb
this is copied from my another repo and was `rtn str`
return was replaced by printf by mistake

Reply:
are you dunk during the coding? 😂